microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -42,13 +42,13 @@ This table lists:
 | Removal of 3DES in TLS | v93 | | Starting with v93, support for the TLS_RSA_WITH_3DES_EDE_CBC_SHA cipher suite will be removed.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/6678134168485888).  Additionally, in v93, a compatibility policy will be available to support scenarios that need to preserve compatibility with outdated servers.  This compatibility policy will become obsolete and stop working in v95.  Make sure that you update affected servers before then. |
-| Deprecate WebRTC's Plan B SDP semantics | v98 (Chrome+2) | | This change is happening in the Chromium project, on which Microsoft Edge is based.  This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B.  This SDP format is being replaced by the Unified Plan, which is a spec-compliant and cross-browser compatible SDP format.  For more information, see the [Chrome Platform Status entry](https://www.chromestatus.com/feature/5823036655665152), [PSA: Plan B should throw in M96 Beta and Stable](https://groups.google.com/g/discuss-webrtc/c/zRIgxG18D80/m/k4ZPzBO3AAAJ), and [PSA: Plan B throwing in Stable and Extended Deprecation Trial End Date](https://groups.google.com/u/1/g/discuss-webrtc/c/gEHrZyYKsfU).  The Microsoft rollout schedule for deprecation is planned for two releases after Chrome.  Requesting a [WebRTC Plan B Reverse Origin Trial Token](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) allows sites to continue to use the deprecated API until v101. |
+| Deprecate WebRTC's Plan B SDP semantics | v104 (Chrome+2) | | This change is happening in the Chromium project, on which Microsoft Edge is based.  This change deprecates a legacy Session Description Protocol (SDP) dialect called Plan B.  This SDP format is being replaced by the Unified Plan, which is a spec-compliant and cross-browser compatible SDP format.  For more information, see the [Chrome Platform Status entry](https://www.chromestatus.com/feature/5823036655665152), [PSA: Plan B should throw in M96 Beta and Stable](https://groups.google.com/g/discuss-webrtc/c/zRIgxG18D80/m/k4ZPzBO3AAAJ), and [PSA: Plan B throwing in Stable and Extended Deprecation Trial End Date](https://groups.google.com/u/1/g/discuss-webrtc/c/gEHrZyYKsfU).  The Microsoft rollout schedule for deprecation is planned for two releases after Chrome.  Requesting a [WebRTC Plan B Reverse Origin Trial Token](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) allows sites to continue to use the deprecated API until v101. |
```

Comment:
@erik-anderson - is "v101" correct at end?

Reply:
@mikehoffms reverted